### DISCUSSION TITLE
List of examples we should probably add to complete the docs

### DISCUSSION THREAD

**Discussion by jgostick** [PMEAL/OpenPNM#2776] [Ideas]
I wrote a quick example notebook last night that we'll be able to point to whenever users ask a question about a 'clustered network' causing problems with the solvers. It occurred to me that we should have a host of these sorts of notebooks which addresses all the common problems people tend to have.  I want to make a list:

- [ ] Adding boundary pores
  - why it's useful but not necessary
  - why it's challenging
- [ ] Calculating porosity
  - We have one on this already but it needs a bit more
  - Calculating the lens volume
- [ ] Problems with conductance calculations
  - Why Dp must be larger than Dt
  - Why it happens
  - How to fix it
- [ ] Matching cubic pnm to data
  - We have something on playing with PSDs
  - We should also talk about matching MIP and K, maybe even Deff
- [ ] Installation options
  - This should maybe be a video
  - Install via conda-forge vs pip
  - Install the dev version
  - Upgrading
- [ ] Explore topotools
  - Look at meaning of stitch, extend, merge, etc
  - Generally explain mangling of networks in some detail

**Comment by jgostick** [comment]
- [ ] How to interpret warnings and exception
  - We issue exceptions in various places, so maybe we should take a survey of these and provide an explanation for what each one means?
  - Similarly for warning messages...we should talk about common ones, their source, and their solution

**Comment by ma-sadeghi** [comment]
- [ ] Simple adsorption process?
    - Show how to implement isotherms as pore-scale models
    - Show how to incorporate the isotherm in the simulation as a source term
